user-service/routes/history.routes.js

```diff
@@ -11,45 +11,18 @@ module.exports = function (app) {
   });
 
   app.post("/api/hist/save", [authJwt.verifyToken], controller.addHistory);
-  // app.post('/api/user/history', [authJwt.verifyToken], controller.addHistory);
 
   app.post(
     "/api/hist/customsave",
     [authJwt.verifyToken],
     controller.addCustomHistory
   );
```

Comment:
One line (same as other routes) might be better :)

```suggestion
  app.post("/api/hist/customsave", [authJwt.verifyToken], controller.addCustomHistory);
```

Reply:
can't bc of auto-formatting when save :(